src/odb/CMakeLists.txt

```diff
@@ -81,3 +81,22 @@ if(ENABLE_TESTS)
     odb
   )
 endif()
+
+add_custom_command(
+  OUTPUT messages.txt
+  COMMAND ${CMAKE_SOURCE_DIR}/etc/find_messages.py > messages.txt && touch ${CMAKE_CURRENT_BINARY_DIR}/messages.txt
```

Comment:
What is the purpose of the touch command?  That file doesn't seem to be referenced.

Reply:
This file is referenced by add_custom_target(odb_messages DEPENDS messages.txt). This file is in build/src/odb and is only generated when find_messages.py succeeds. This way invoking make will rerun find_messages.py if it's missing or if a dependencies is changed.
The output of find_messages.py is in src/odb and is always generated whenever find_messages.py succeeds or not which is the current behaviour.